CONTRIBUTING.md

```diff
@@ -36,6 +36,18 @@ Contributor Agreement (ECA) on file.
 For more information, please see the Eclipse Committer Handbook:
 https://www.eclipse.org/projects/handbook/#resources-commit
 
+## Project Contribution Guidelines
+
+ * Code should adhere to the style defined at https://github.com/eclipse-ee4j/ee4j.
+ * Contributions should be made via github pull requests at https://github.com/eclipse-ee4j/servlet-api/pulls 
+ * Pull Requests should have reviews requested from several project committers from different organisations than the author(s). 
```

Comment:
Without knowing the availability and current commitments of other committers I'm not sure how I'd identify appropriate committers to assign to the review. I'm also concerned that it may encourage other committers not to review the PR because someone else has been assigned. Finally, I think the "different organisations" part could be read as "we don't trust committers with the same employer not to collude to the advantage of their employer". I'm wondering whether the following point is sufficient to make this one unnecessary? Or maybe explicitly state that reviewing PRs is the responsibility of all committers but we also welcome constructive reviews from non-committers?

Reply:
I can see how that reads negatively, but it is really meant as a positive guideline to seek review from outside ones own bubble.   It wasn't really meant to be about the mechanics of requesting a review.
How about breaking it into two points:
+ Authors of Pull Requests should seek review from committers outside of their own organisation and perspective.
+ Committers and Contributors may review any Pull Requests, even if not requested or assigned.

Reply:
That works for me